.github/ISSUE_TEMPLATE/vetsapi-argo-terminal-access.yaml

```diff
@@ -0,0 +1,132 @@
+        2. Change the **Title** to include target individual
+        3. Add the label used by the target individual's team (eg: `BAH-526`)
+        4. Do not remove any of the existing labels.
+        5. Prod access is highly restricted. If you need prod access, add an explanation in "Additional Notes" section.
+        6. Validate that your E-QIP process has been started with your Vendor Onboarding Representative (listed below in a dropdown). If you have not started your E-QIP process, do NOT put in a request.
+  - type: checkboxes
+    attributes:
+      label: E-QIP Transmittal Date
+      description: Have you started your E-QIP process? If not, do not submit this ticket. Ask your Vendor Onboarding Representative to confirm
+      options:
+      - label: Check this box if you have an E-QIP transmittal date.
+    validations:
+      required: true
+  - type: input
+    id: requestor-name
+    attributes:
+      label: Your Name
+      description: Please provide the name of the target individual
+      placeholder: Jane Doe
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Your GitHub handle (terminal access is managed through GitHub)
+      description: Please provide the work GitHub handle of the target individual
+      placeholder: "@username"
+    validations:
+      required: true
+  - type: input
+    id: team-name
+    attributes:
+      label: Team, Role, and Company of the target individual
+      description: Please provide the name of the team, their role on that team, and the name of the company the target individual works for
+      placeholder: Team Moose, Antler Development, Alces LLC
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Product Manager (PM) name and email
+      description: Provide the name and email of the Product Manager working with the target individual
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Product Owner (PO) name and email
+      description: Provide the name and email of the Product Owner for the target individual
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Your dsvagovcloud AWS user name (if applicable)
+      description: If you have previously been granted AWS access, list your dsvagovcloud AWS user name
+      placeholder: First.Last
+  - type: checkboxes
+    attributes:
+      label: ArgoCD terminal access requested for the following environments
```

Comment:
Does this sound normal?? I couldn't figure out how to phrase it lol

Reply:
LGTM